assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "a77f4d07-c6e0-4a48-8b35-0eeb51576f4f",
+  "queryName": "Allways Pull Images Admission Control Plugin Not Set",
```

```suggestion
  "queryName": "Always Pull Images Admission Control Plugin Not Set",
```

Please, also update the query folder name